startupscript/dataproc/startup.sh

```diff
@@ -935,6 +947,58 @@ fi
     ${RUN_PYTHON} ${HAIL_SCRIPT_PATH}
   fi
 
+  #################################
+  # Configure Proxy Agent Overrides
+  #################################
+
+  # Map the CLI server to the appropriate UI url
+  case $TERRA_SERVER in
+    verily-devel)
+      ui_base_url=terra-devel-ui-terra.api.verily.com
+      ;;
+    verily-autopush)
+      ui_base_url=terra-autopush-ui-terra.api.verily.com
+      ;;
+    verily-staging)
+      ui_base_url=terra-staging-ui-terra.api.verily.com
+      ;;
+    verily-preprod)
+      ui_base_url=terra-preprod-ui-terra.api.verily.com
+    ;;
+    verily)
+      ui_base_url=workbench.verily.com
+      ;;
+    *)
+      >&2 echo "ERROR: $TERRA_SERVER is not a known verily server."
+      exit 1
+      ;;
+  esac
+
+  # Insert a link element to render
+  readonly WORKSPACE_LINK_EL="<a id=\"workspace\" class=\"forum\" target=\"_blank\" href=\"https://${ui_base_url}/workspaces/${TERRA_WORKSPACE}\">${TERRA_WORKSPACE}</a>"
```

Comment:
I think this reads easier as:

```
  readonly WORKSPACE_LINK_EL='<a id="workspace" class="forum" target="_blank" href="https://'"${ui_base_url}/workspaces/${TERRA_WORKSPACE}"'"'">${TERRA_WORKSPACE}</a>"
```

Though admittedly getting that last double-quote around the href is a little clunky.